src/plugins/orangehrmRecruitmentPlugin/entity/Decorator/InterviewDecorator.php

```diff
@@ -82,4 +82,22 @@ private function addInterviewer(Employee $employee): void
         }
         $interviewers[] = $employee;
     }
+
+    /**
+     * @return string
+     */
+    public function getInterviewDate(): string
+    {
+        $interviewDate = $this->interview->getInterviewDate();
+        return $interviewDate->format('Y-m-d');
```

Comment:
```suggestion
        return $this->getDateTimeHelper()->formatDateTimeToYmd($interviewDate);
```
can use date tile helper also

Reply:
done